config/initializers/spotlight_initializer.rb

```diff
@@ -116,7 +116,7 @@
 #   'my-local-config': ->(context) { context.my_custom_data_path(context.current_exhibit) }
 # }
 
-default_themes = %w[default example]
+default_themes = %w[default example rmc-legacy]
```

Comment:
Since this theme will be available for all exhibits, might be worth mentioning it in the Confluence curator docs: https://confluence.cornell.edu/display/exhibits/Custom+Visual+Theme

Reply:
If we don't want it available for all exhibits, you can update this initializer code so that this theme is only available for certain exhibit slugs.

Reply:
There may be a future where we have many themes available for exhibit admins to choose from. We can revisit which ones should be available to all exhibits and which should be exhibit-specific then.

As long as you announce this new available theme in the Developer Release Notes after this is deployed, that should be fine.